frappe/tests/ui_test_helpers.py

```diff
@@ -136,13 +136,14 @@ def create_contact_records():
 
 @frappe.whitelist()
 def create_multiple_todo_records():
+	values = []
 	if frappe.db.get_all('ToDo', {'description': 'Multiple ToDo 1'}):
 		return
-	for index in range(501):
-		frappe.get_doc({
-			'doctype': 'ToDo',
-			'description': 'Multiple ToDo {}'.format(index+1)
-		}).insert()
+
+	for index in range(1, 1002):
+		values.append(('100{}'.format(index), 'Multiple ToDo {}'.format(index)))
```

Comment:
Although this is a lot better than the previous, `values` could be a single list comprehension 😅 .

This whole block could look a whole lot simpler. Since you generated a list of unique tuples, there isn't any way they could be any duplication - I'm not sure why you've casted the list to a set before passing it to `bulk_insert`

Reply:
Yeah, I should have used list comprehension my bad 😅. I will fix this👍